sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -659,6 +659,11 @@ public static CosmosAsyncContainer getControlContainerFromThroughputGlobalContro
         return globalControlConfig.getControlContainer();
     }
 
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static int getNumberOfRegionsToTry(AvailabilityStrategy availabilityStrategy) {
```

Comment:
let's not add it - instead use `ImplementationBridgeHelpers`

Reply:
removed